src/components/NavigationBar/SignOutButton.tsx

```diff
@@ -2,20 +2,14 @@
 
 import React from "react";
 import LogoutIcon from "@mui/icons-material/Logout";
-import { DatabaseAutoType } from "@/databaseUtils";
-import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import IconButton from "@mui/material/IconButton/IconButton";
 
-const SignOutButton: React.FC<{}> = () => {
-    const supabase = createClientComponentClient<DatabaseAutoType>();
-
+interface Props {
+    onClick: () => void;
+}
+const SignOutButton: React.FC<Props> = (props: Props) => {
```

Comment:
let's add an empty line above for separation
```suggestion

const SignOutButton: React.FC<Props> = (props: Props) => {
```